### DISCUSSION TITLE
How many logs in memory you can have while the TCP server is down?

### DISCUSSION THREAD

**Discussion by GabrielJuliao** [logfellow/logstash-logback-encoder#980] [General]
Hi All! I know when you are using a TCP appender, if the server receiving the logs is down when it eventually becomes available the appender will flush the logs that were pending. So, how many logs can I have? Am I going to lose logs if the server stays down long enough? Is there a buffer if so, what's the size?

**Comment by philsttr** [comment]
See the [RingBuffer Size](https://github.com/logfellow/logstash-logback-encoder#ringbuffer-size) and [RingBuffer Full](https://github.com/logfellow/logstash-logback-encoder#ringbuffer-full) sections of the readme.

**Reply by GabrielJuliao** [reply]
What 8192 means? Whats the unit? I've done some tests I was able to get 332KB of logs, and it seems that I could get more.

**Reply by philsttr** [reply]
8192 log events
